src/OpenProject/ViewModels/WebViewModel.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows;
+using CefSharp.Wpf;
+using OpenProject.Api;
+using OpenProject.Data.Utils;
+using OpenProject.WebViewIntegration;
+using OpenProject.Windows;
+
+namespace OpenProject.ViewModels
+{
+  public sealed class WebViewModel
+  {
+    private readonly JavaScriptBridge _javaScriptBridge;
+    private readonly BrowserManager _browserManager;
+
+    public ChromiumWebBrowser Browser => _browserManager.Browser;
+
+    public WebViewModel(BrowserManager browserManager, JavaScriptBridge javaScriptBridge)
+    {
+      _browserManager = browserManager;
+      _javaScriptBridge = javaScriptBridge;
+
+      _browserManager.Initialize();
+      InitializeIpcConnection();
+
+      if (UserSettings.GetBool("checkupdates"))
+        CheckUpdates();
```

Comment:
I'm always anxious over missing curly braces😀 It reminds me of the Apple Curly Braces bug a few years ago.

Reply:
I know this anxiety :D For really simple one-liners like a method call or a `return;` I nevertheless like it to drop the curlies for readability's sake.

I know of the C bug with curlys, but not the mentioned apple bug. You have a link?

Reply:
Essentially this error:
![image](https://user-images.githubusercontent.com/10274404/127626672-3287e9d8-f696-4cbe-97aa-ae00bbe00805.png)

Afair, it came from some merge conflict, and disabled SSL validation on millions of devices.

https://embeddedgurus.com/barr-code/2014/03/apples-gotofail-ssl-security-bug-was-easily-preventable/

But there are lots of Google hits for the bug, not sure which article is the best. Just remember it from some years ago.

Reply:
Ah, I see. Well, when it comes from merges, this is really fucked up and can only be solved by correct automated tests. When coding directly in an IDE, this cannot happen, as the IDE directly puts a warning on the indentation.

Interesting story though :D as a non-apple user, I didn't even know about this bug.

Reply:
>this cannot happen

Famous last words😀

Reply:
You know, I like it pretentious, just before failing (with grace). :D